src/views/CooperationInfo.vue

```diff
@@ -187,30 +193,158 @@
               :closable="false"
               :dismissableMask="true"
             >
-              <form @submit.prevent="editHouseInfo">
+              <form submit.prevent="editHouseInfo">
                 <p>
                   <label class="dialog-item" for="coopName">Кількість квартир в будинку : </label>
-                  <InputText id="quantityFlat" placeholder="Кількість квартир в будинку" v-model="quantity_flat" />
+                  <InputText
+                    id="quantityFlat"
+                    placeholder="Кількість квартир в будинку"
+                    v-model.trim="house.quantity_flat"
+                    :class="{ 'p-invalid': v$.house.quantity_flat.$error }"
+                    @blur="v$.house.quantity_flat.$touch"
+                  />
+                  <small v-if="v$.house.quantity_flat.$error" class="p-error">{{
+                    v$.house.quantity_flat.$errors[0].$message
+                  }}</small>
                 </p>
                 <p>
                   <label class="dialog-item" for="coopAddress">Площа будинку : </label>
-                  <InputText id="houseArea" placeholder="Площа будинку" v-model="house_area" />
+                  <InputText
+                    id="houseArea"
+                    placeholder="Площа будинку"
+                    v-model="house.house_area"
+                    :class="{ 'p-invalid': v$.house.house_area.$error }"
+                    @blur="v$.house.house_area.$touch"
+                  />
+                  <small v-if="v$.house.house_area.$error" class="p-error">{{
+                    v$.house.house_area.$errors[0].$message
+                  }}</small>
                 </p>
                 <p>
                   <label class="dialog-item" for="iban">Прибудинкової теріторії : </label>
-                  <InputText id="adjoiningArea" placeholder="Прибудинкової теріторії" v-model="adjoining_area" />
+                  <InputText
+                    id="adjoiningArea"
+                    v-model="house.adjoining_area"
+                    :class="{ 'p-invalid': v$.house.adjoining_area.$error }"
+                    @blur="v$.house.adjoining_area.$touch"
+                  />
+                  <small v-if="v$.house.adjoining_area.$error" class="p-error">{{
+                    v$.house.adjoining_area.$errors[0].$message
+                  }}</small>
+                </p>
+                <p>
+                  <label class="dialog-item" for="coopEmail">
+                    <pre>Адреса:</pre>
+                    <pre>     Регіон :</pre>
```

Comment:
It's better to make this whitespaces in styles, but not with spaces in the element

Reply:
removed spaces and added styles for proper layout